### DISCUSSION TITLE
Role Mappings are missing in new admin console

### DISCUSSION THREAD

**Discussion by Agh42** [keycloak/keycloak#14954] [Archive]
We just upgraded to v 19.0.3 and to the new Admin Console. When I try to edit _"Groups -> MyGroup -> Group Details -> Role Mapping"_, no realm- or client-roles are shown. In our setup we exclusively use groups to assign roles to users.

The assigned roles are still present in the access token when I examine it via _"Clients -> MyClient -> Client Scopes -> Evaulate"_. When I switch back to the v1 Admin Console theme, I can still see and edit all role mappings in my groups.

Is this a known issue? Do I need to do any active migration when moving to the v2 theme?

Theme v1:
![image](https://user-images.githubusercontent.com/7584303/196372768-e9cff391-7f8a-42d7-bd1b-7eb5c0cf704e.png)

Theme v2:
![image](https://user-images.githubusercontent.com/7584303/196374520-37608e1e-d692-4baa-bba8-e5a8105a7549.png)

**Comment by oleeander** [comment]
I've verified that this only affects child groups. If you remove the UUID of the parent from the URI it works:
```patch
-http://keycloak-ui/admin/master/console/#/example-realm/groups/21f71205-ecf2-4ed3-85c1-e9817a6a69a5/a1825189-997d-4f5d-b6b0-56807c6f0351
+http://keycloak-ui/admin/master/console/#/example-realm/groups/a1825189-997d-4f5d-b6b0-56807c6f0351
```
That issue is tracked here: https://github.com/keycloak/keycloak-ui/issues/3515

**Reply by Agh42** [reply]
Thanks for the update - that gives me an acceptable workaround by simply changing the structure of our groups for now!

**Comment by ssilvert** [comment]
@oleeander @Agh42 There is reason to believe that this is already fixed in the nightly build.  It would be a great help if you could try it out and report your results in https://github.com/keycloak/keycloak-ui/issues/3515.

Thanks.

https://github.com/keycloak/keycloak/releases/tag/nightly
